packages/services/pkg/protocol-parser/decodeDynamicField.go

```diff
@@ -0,0 +1,137 @@
+package protocolparser
+
+import (
+	. "latticexyz/mud/packages/services/pkg/schema-type"
+
+	"github.com/andriidski/abiencode-go/convert"
```

Comment:
this seems to be a private package? would it be possible to vendor it in this repo?